pyproject.toml

```diff
@@ -62,6 +60,8 @@ test = [
     "pytest-cov==6.0.0",
     "pytest-rerunfailures==15.0",
     "xkcdpass==1.19.9",
+    "types-requests==2.32.0.20241016",
+    "types-setuptools==75.8.0.20250210",
```

Comment:
Thanks for moving them out of the runtime dependencies.

May I request to move these 1 line up so that they are ordered alphabetically? :)